tests/integ/modin/conftest.py

```diff
@@ -224,16 +225,19 @@ def indices_dict():
         # NumericIndex is a pandas 2.x feature
         "num_int32": pd.Index(np.arange(INDEX_SAMPLE_SIZE), dtype="int32"),
         "num_int16": pd.Index(np.arange(INDEX_SAMPLE_SIZE), dtype="int16"),
-        "num_int8": pd.Index(np.arange(INDEX_SAMPLE_SIZE)).astype("int8"),
+        # TODO: SNOW-1372242: Remove instances of to_pandas when lazy index is implemented
+        "num_int8": pd.Index(np.arange(INDEX_SAMPLE_SIZE)).to_pandas().astype("int8"),
```

Comment:
Shouldn't these be converted back to a Modin index? The other indices in this dict are all Modin indices.

Reply:
what about change all pd.Index to native_pd.Index?

Reply:
I can do that, is that better?